storage/test/tuples.go

```diff
@@ -457,3 +458,59 @@ func TuplePaginationOptionsTest(t *testing.T, datastore storage.OpenFGADatastore
 		}
 	})
 }
+
+func ReverseReadTuplesTest(t *testing.T, datastore storage.OpenFGADatastore) {
+
+	require := require.New(t)
```

Comment:
This is a stupid simple test. We may want to add more, but this outlines the gist of it.